### PR TITLE
Hidden the breadcrumbs block

### LINKED ISSUE #111: Remove or hide breadcrumbs on pages

### PR DESCRIPTION
**Pull request by ajtrichards**
This closes #111

### CODE REVIEW

File: docs/local-development.md [hunk 1]
```diff
@@ -28,4 +28,6 @@
     fin drupal-install
     ```
 
+    If this stage fails due to `RuntimeException: Missing $settings['hash_salt'] in settings.php` - make sure you have the `.env` file in your root folder and there's a random string for the ENV variable `DRUPAL_SALT`
```
Comment (opdavies) [inline docs/local-development.md]: @ajtrichards Can this be moved into a separate PR?

Comment (ajtrichards) [reply docs/local-development.md]: Yeah no problem 👍

**Comment by opdavies** [comment]
Merged! Thanks for the contribution, @ajtrichards.
